### PR TITLE
[docs] Update for 2.1.6 release, including links and whats new

### PR DESCRIPTION
**Pull request by stevebang**

### CODE REVIEW

File: docs/content/latest/releases/earlier-releases/v2.1.5.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: What's new in 2.1.5
```
Comment (schoudhury) [inline docs/content/latest/releases/earlier-releases/v2.1.5.md:2]: remove what's new from this page's front matter

File: docs/content/latest/releases/whats-new/_index.md [hunk 2]
```diff
@@ -41,58 +40,38 @@ menu:
...
-- [YW] Add **IAM Role** toggle in provider storage configuration to use the IAM role instead of requiring an **Access Key** and **Secret**. [#4204](https://github.com/yugabyte/yugabyte-db/issues/4204)
-- [YW] When creating a universe and AWS provider is selected, display new **Use IAM Profile** toggle and **ARN String** text field. [#4199](https://github.com/yugabyte/yugabyte-db/issues/4199)
-- Use Raft configuration as a source for master addresses in `server.conf` for master. [#4089](https://github.com/yugabyte/yugabyte-db/issues/4089)
-- [YW] If a node appears as unreachable, it can be removed or released without generating errors. [#4171](https://github.com/yugabyte/yugabyte-db/issues/4171)
-- [YW] Create GCP providers with any combination of host credentials and host (or shared) VPC. [#4177](https://github.com/yugabyte/yugabyte-db/issues/4177)
+- When performing a full move or add node on a universe that has a yb-master, the `server.conf` file is now being updated with the new `master_addresses`. [#4242](https://github.com/yugabyte/yugabyte-db/issues/4242)
+
+### YugabyteDB Admin Console
```
Comment (schoudhury) [inline docs/content/latest/releases/whats-new/_index.md:68]: this section header should be removed and all these things should be listed only under Yugabyte Platform.

File: docs/content/latest/releases/earlier-releases/v2.1.5.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+headerTitle: What's new in 2.1.5
+linkTitle: What's new in 2.1.5
+description: Enhancements, changes, and resolved issues in the latest YugabyteDB release.
```
Comment (schoudhury) [inline docs/content/latest/releases/earlier-releases/v2.1.5.md:5]: this description has to change as well to match that this is now a specific earlier release (similar to how other earlier releases are handled)

Comment (stevebang) [reply docs/content/latest/releases/earlier-releases/v2.1.5.md:5]: Fixed.

File: docs/content/latest/releases/earlier-releases/v2.1.5.md (hunk 1 above)
Comment (stevebang) [reply docs/content/latest/releases/earlier-releases/v2.1.5.md:2]: Fixed

File: docs/content/latest/releases/whats-new/_index.md (hunk 2 above)
Comment (stevebang) [reply docs/content/latest/releases/whats-new/_index.md]: Fixed

**Pull request merged**